applications/luci-app-ntpc/htdocs/luci-static/resources/view/ntpc.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+'require form';
+'require rpc';
+'require view';
+
+var callSystemInfo = rpc.declare({
+	object: 'system',
+	method: 'info'
+});
+
+return view.extend({
+	load: function () {
+		return Promise.all([
```

Comment:
Here you have only one promise so you can remove the Promise.all()

Reply:
Fixed